tests/UI/pages/BO/catalog/discounts/add.js

```diff
@@ -35,17 +39,21 @@ class AddCartRule extends BOBasePage {
     // Fill information form
     await this.setValue(page, this.nameInput(1), cartRuleData.name);
     await this.setValue(page, this.descriptionTextArea, cartRuleData.description);
-    await this.setValue(page, this.codeInput, cartRuleData.code);
+    if (cartRuleData.code) await this.setValue(page, this.codeInput, cartRuleData.code);
+    else await page.click(this.generateButton);
```

Comment:
```suggestion
    if (cartRuleData.code) {
      await this.setValue(page, this.codeInput, cartRuleData.code);
    } else {
    await page.click(this.generateButton);
    }
```

Reply:
Why not adding an eslint job for this? :thinking:

Reply:
Ping @NeOMakinG @boubkerbribri

Reply:
https://eslint.org/docs/rules/curly

Agree

Reply:
Also, a github action!

Reply:
An eslint configuration already exists for the tests/UI project :D

Reply:
> Also, a github action, if it doesn't exist on develop

It can be added to the workflow already existing in develop (does not exist yet), I will work on that.

> Why not adding an eslint job for this? thinking

We have already a eslint --fix command that should fix that rule, but it don't, I will try to find out why.

Reply:
Done thx :+1: